setup-gcloud/src/setup-gcloud.ts

```diff
@@ -87,17 +108,15 @@ async function run(): Promise<void> {
     });
     await fs.writeFile(tmpKeyFilePath, serviceAccountJSON);
 
-    // A workaround for https://github.com/actions/toolkit/issues/229
-    // Currently exec on windows runs as cmd shell.
-    let toolCommand = 'gcloud';
-    if (process.platform == 'win32') {
-      toolCommand = 'gcloud.cmd';
-    }
-
-    // authenticate as the specified service account
-    await exec.exec(
-      `${toolCommand} auth activate-service-account ${serviceAccountEmail} --key-file=${tmpKeyFilePath}`,
```

Comment:
So, fun fact (for some definition of "fun"), we weren't escaping any of these values, so you can RCE yourself 😄

Reply:
Good catch!